lib/buffer.js

```diff
@@ -351,7 +364,7 @@ Buffer.concat = function(list, length) {
     length = length >>> 0;
   }
 
-  var buffer = Buffer.allocUnsafe(length);
+  var buffer = allocate(length);
```

Comment:
Just to make sure, the `s/Buffer.allocUnsafe/allocate/` changes here are completely unrelated to the rest of the changes, right?